### ISSUE TITLE
Class constructor Subject cannot be invoked without 'new'

### ISSUE THREAD

**Issue by brunojcm** [HNeukermans/ng2-signalr#139]
I'm trying to migrate my app from Angular 7 to Angular 8, `ng2-signalr` is the last piece in my way. 

The error I get is the following:

```
ERROR TypeError: Class constructor Subject cannot be invoked without 'new'
    at new BroadcastEventListener (vendor.js:198620)
    at ServerNotificationService.subscribeToEventType (main.js:7977)
    at new ServerNotificationService (main.js:7965)
    at vendor.js:38803
    at _callFactory (vendor.js:60799)
    at _createProviderInstance (vendor.js:60742)
    at resolveNgModuleDep (vendor.js:60701)
    at NgModuleRef_.get (vendor.js:61794)
    at resolveDep (vendor.js:62324)
    at createClass (vendor.js:62168)
```

`ServerNotificationService.subscribeToEventType` is doing something like:

```
    let onDismiss$ = new BroadcastEventListener<string>('dismissNotification')
    this._signalRConnection.listen(onDismiss$)
    onDismiss$.subscribe((guid) => {
      this.dismiss(guid, true)
    })
```

As Angular 8 defaults to `es2015` and differential loading (https://www.i-programmer.info/news/167-javascript/12815-angular-8-adds-default-differential-loading-.html), we should probably have some work to adjust this library to work on this scenario.

Any thoughts?

**Comment by Franki1986** [comment]
I have the same error!

**Comment by HMubaireek** [comment]
Same error here! Temporary solution is to change "target": "es5" in tsconfig.

**Comment by manojdemel12d** [comment]
I have the same error. Setting target to "**es5**" fixes the issue as mentioned by @HMubaireek

**Comment by thduttonuk** [comment]
I have the same error on Angular 8, will look to fix

**Comment by lattimoregay** [comment]
For me changing to es5 is not an option, so curious when this may be addressed in the library for later targets ?

**Comment by thduttonuk** [comment]
@lattimoregay I have a fix going out, currently 8.0.1-beta1 works for me but I have some testing and cleanup to do before final release.

**Comment by lattimoregay** [comment]
Great. Do you have an eta on this.

**Comment by thduttonuk** [comment]
I have pushed out the package so you can use Beta now. Can you let me know if you have issues? Its working for my project using differential loading.

**Comment by thduttonuk** [comment]
Released 8.0.2
